<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vtnPrincipal</class>
 <widget class="QMainWindow" name="vtnPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sistema de Gestion del Personal</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnNuevo">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>320</y>
      <width>75</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Nuevo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLimpiar">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>320</y>
      <width>75</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Limpiar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBorrarRegistro">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>320</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Borrar Registro</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnActualizar">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>320</y>
      <width>78</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Actualizar</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>358</width>
      <height>26</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
    <widget class="QLabel" name="lbBuscarUsuario">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Buscar por Usuario:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtBuscarUsuario">
     <property name="maxLength">
      <number>20</number>
     </property>
    </widget>
    <widget class="QPushButton" name="btnBuscarUsuario">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::EditFind"/>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="cbUsuario">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>210</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Seleccionar</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Administrador</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Operador</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Invitado</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lbUsuario">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>204</y>
      <width>79</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Usuario:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbClave">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>177</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Clave:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbUsuarioid">
    <property name="geometry">
     <rect>
      <x>88</x>
      <y>150</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>UsuarioID:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtUsuarioid">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtClave">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>706</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
